app/code/Magento/InventoryBundleProduct/Test/Integration/Order/PlaceOrderOnDefaultStockTest.php

```diff
@@ -138,15 +140,6 @@ public function testPlaceOrderWithOutOfStockProductAndBackOrdersTurnedOn()
         $bundleProduct = $this->productRepository->get($bundleSku);
 
         $cart->addProduct($bundleProduct, $this->getBuyRequest($bundleProduct, $qty));
```

Comment:
after your changes, some tests started to fail because of lack of preconditions, but just removing order placement - we are losing the whole order placement scenarios for composite product types. 
Thus, along with coverage of what you did (now tests cover these scenarios) you need to provide pre-conditions for existing tests to pass Order placement scenarios for composite product types